cg/cli/workflow/balsamic/base.py

```diff
@@ -67,6 +68,7 @@ def balsamic(context, priority, panel_bed, analysis_type, run_analysis, dry):
         housekeeper_api=HousekeeperAPI(config),
         fastq_handler=FastqHandler(config),
         lims_api=LimsAPI(config),
+        trailblazer_api=TrailblazerAPI(config),
```

Comment:
Why do we send in whole config object everywhere 😞 I don't think you should change it here of course but it should be changed...